### PR TITLE
i added i love bulb file and edited readme file

### PR DESCRIPTION
**Pull request by chilara**
i have changed my stuffs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # learnCode
 learning git
 
-I love coding
\ No newline at end of file
+I love coding and graphics designing
```
Comment (andrewinsoul) [inline README.md]: Remove design and put only graphics

**Review by andrewinsoul — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chilara) [reply README.md]: ok thanks

Comment (chilara) [reply README.md]: done

Comment (andrewinsoul) [reply README.md]: Great

**Review by andrewinsoul — APPROVED** [review]

**Pull request merged**